Cython/Utility/ModuleSetupCode.c

```diff
@@ -2190,17 +2214,25 @@ static PyObject* __Pyx_PyCode_New(
     }
 
     #if CYTHON_COMPILING_IN_LIMITED_API
+    #if CYTHON_AVOID_UNSAFE_BORROWED_REFERENCES && PY_VERSION_HEX > 0x030d00b1
+    PyDict_GetItemRef(tuple_dedup_map, varnames_tuple, &varnames_tuple_dedup);
```

Comment:
These aren't quite the same from an exception handling point-of-view.

Reply:
That's true. `PyDict_GetItemRef` raises an exception on error. What would you prefer we do in this case? One option would be:

```cpp
#if CYTHON_AVOID_UNSAFE_BORROWED_REFERENCES && __PYX_LIMITED_VERSION_HEX > 0x030d00b1
int res = PyDict_GetItemRef(tuple_dedup_map, varnames_tuple, &varnames_tuple_dedup);
if (res == -1) goto done;
if (res == 0) {
#else
varnames_tuple_dedup = PyDict_GetItem(tuple_dedup_map, varnames_tuple);
if (!varnames_tuple_dedup) {
#endif
```